test_utils/src/lib.rs

```diff
@@ -268,6 +268,28 @@ pub fn fixture_header_builder() -> types::HeaderBuilder<Ed25519PublicKey> {
         )
 }
 
+pub fn fixture_headers_round(
+    prior_round: Round,
+    parents: &BTreeSet<CertificateDigest>,
+) -> (Round, Vec<Header<Ed25519PublicKey>>) {
```

Comment:
This takes parents at round `prior_round` and constructs the next round of headers (using all keys in test). To be used with the `certificate` helper function in this same file for generating certs.